### PR TITLE
London Class 7 - Joy Omodiale - JavaScript Core 1 - Week3

### PR DESCRIPTION
**Pull request by jmodiale**
# Your Details

- Your Name: Joy Omodiale
- Your City: London
- Your Slack Name:

# Homework Details

- Module: JavaScript-Core-1-Coursework
- Week: Week3

### CODE REVIEW

File: 1-exercises/H-array-methods-2/exercise.js [hunk 1]
```diff
@@ -15,8 +15,8 @@ var everyone = [
   "Swathi",
 ];
 
-var firstFive; // complete this statement
-var lastFive; // complete this statement
+var firstFive = everyone.slice(0, 5); // complete this statement
+var lastFive = everyone.slice(2, 7); // complete this statement
```
Comment (aishak05) [inline 1-exercises/H-array-methods-2/exercise.js:19]: Hi Joy, great work. Just an alternative way of getting the last 5 could be using .slice(-5)

File: 1-exercises/A-array-find/exercise.js [hunk 2]
```diff
@@ -17,7 +17,11 @@ var names = [
   "Ahmed",
 ];
 
-var longNameThatStartsWithA = findLongNameThatStartsWithA(names);
+function findLongNameThatStartsWithA(name){
+  return name.startsWith('A') && name.length > 7;
+}
```
Comment (ybayraktutan) [inline 1-exercises/A-array-find/exercise.js:22]: Hi Joy, well done for using a new method ".startsWith('A')"

File: 2-mandatory/7-password-validator.js [hunk 3]
```diff
@@ -24,7 +24,22 @@ PasswordValidationResult=  [false, false, false, false, true]
...
+      /[!#$%.*&]/.test(password) &&
+      array.indexOf(password) === index
+    ) {
+      return true;
+    } else {
+      return false;
+    }
+  });
+  return thePass
 }
```
Comment (ybayraktutan) [inline 2-mandatory/7-password-validator.js:43]: You have combined all in one function. Great work.

**Review by ybayraktutan — COMMENTED** [review]
Hi Laura. Well done, great job. I learned new tips by reviewing your code! I have only a few suggestions to improve it.

File: 1-exercises/H-array-methods-2/exercise.js (hunk 1 above)
Comment (jmodiale) [reply 1-exercises/H-array-methods-2/exercise.js:19]: Thanks Aisha

**Pull request closed without merging**